2.4-micro/root/usr/share/container-scripts/httpd/common.sh

```diff
@@ -24,13 +24,19 @@ gen_ssl_certs() {
 
   echo "---> Generating SSL key pair for httpd..."
   if [ -x "/usr/bin/sscg" ]; then
+      if sscg --help | grep -q dhparams-file; then
```

Comment:
Don't see any problem. Maybe only this indentation :).

Reply:
Thanks @uhliarik - pushed a fix.